src/main/java/com/adobe/target/edge/client/service/NotificationService.java

```diff
@@ -74,4 +76,11 @@ public void buildNotifications(
             .build();
     this.sendNotification(notificationRequest);
   }
+
+  public static void setBeaconToFalse(DeliveryRequest deliveryRequest) {
+    Context context = deliveryRequest.getContext();
+    if (context != null && context.getBeacon() != Boolean.valueOf(false)) {
```

Comment:
`context` is always not null, but `context.getBeacon()` may be null and we'll get an NPE in this case.
```suggestion
    if (context.getBeacon() != null && context.getBeacon()) {
```

Reply:
@XDex - I'm not sure it's likely that user would set beacon to null - but it is possible.  So I wanted to cover that edge case and updated it to:
`if (context.getBeacon() == null || context.getBeacon()) {`